pkg/pluginmanager/manager.go

```diff
@@ -733,7 +737,17 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 	}
 
 	// Log message based on different installation conditions
-	logPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
+	installingMsg, installedMsg := getPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
+
+	var spinnerErr error
+	var spinner component.OutputWriterSpinner
+	if enableSpinner {
+		// Initialize the spinner
+		spinner, spinnerErr = component.NewOutputWriterSpinnerWithOptions(os.Stdout, "text", installingMsg, true, nil)
+		if spinnerErr != nil {
+			log.Warning("Unable to initialize spinner")
```

Comment:
If for some reason spinner initialization fails, I think it would be better to just show a static log message instead of showing this warning message which might be confusing for the user. 

And we should show the warning message at a higher log level. `log.V(6).Warningf("Unable to initialize spinner. Err: %v", err)`.

Reply:
Agreed, fixed.

Reply:
@chandrareddyp You can add the `log.V(6)` message suggested by Anuj as an extra printout

Reply:
Yupdated.